src/main/resources/sql/payment_terms/dev-pg-pt-drop.sql

```diff
@@ -0,0 +1 @@
+drop table if exists payment_terms cascade;
```

Comment:
do not commit this file as part of your final commit. The scripts should be moved to other project.